### ISSUE TITLE
Rejecting cookies takes you to Wikipathways

### ISSUE THREAD

**Issue by AlasdairGray** [ammar257ammar/snorql-extended#6]

**Comment by ammar257ammar** [comment]
Fixed.
It now redirects to google.com instead.
Do you have some other response in mind when rejecting the cookie policy?
Thank you.

**Comment by AlasdairGray** [comment]
I don't think that is the answer either. Perhaps as part of the deployment parameters a suitable location/page can be specified.

At the moment the pop-up makes it nice and easy to reject, but doesn't really specify why you would want to do one or the other. Perhaps you can make it clear that you don't use 3rd party cookies.

**Comment by marvinm2** [comment]
Closing as obsolete. The cookie consent banner was removed in commit a8db9e9 ("Remove cookie policy — no functional cookies remain"), so there is no longer a reject-cookies action that could redirect anywhere. No functional cookies remain in the application.
